extension/src/test/fixtures/expShow/base/customPlots.ts

```diff
@@ -35,6 +35,18 @@ export const customPlotsOrderFixture: CustomPlotsOrderValue[] = [
 ]
 
 export const experimentsWithCheckpoints: ExperimentWithCheckpoints[] = [
```

Comment:
[Q] Is this still the best name?

Reply:
I think so since the actual function that we get the experiments from is called `getExperimentsWithCheckpoints` 🤔  Unless, I'm missing something?